example/linux/Makefile

```diff
@@ -5,6 +5,19 @@ all: build codesign
 codesign:
 	codesign --entitlements vz.entitlements -s - ./virtualization
 
-.PHONY: build
+.PHONY: build cross-build
 build:
 	go build -o virtualization .
+
+cross-build: virtualization-amd64 virtualization-arm64
+
+virtualization-amd64 virtualization-arm64: virtualization-%: force-build
+	CGO_ENABLED=1 GOOS=darwin GOARCH=$* go build -o $@ .
+
+# the go compiler is doing a good job at not rebuilding unchanged files
+# this phony target ensures out/vfkit-* are always considered out of date
+# and rebuilt. If the code was unchanged, go won't rebuild anything so that's
+# fast. Forcing the rebuild ensure we rebuild when needed, ie when the source code
+# changed, without adding explicit dependencies to the go files/go.mod
+.PHONY: force-build
+force-build:
```

Comment:
```suggestion
```

Reply:
Without this, `make` won't work as expected. 
```
$ cd example/linux
$ make
go build -o virtualization .

# make changes to main.go

$ make
# `virtualization` is not rebuilt because there is already a binary with that name
```

This `force-build` target ensures that `go build` is always run for the `virtualization` target. Maybe `.PHONY` would actually achieve the same